.editorconfig

```diff
@@ -1,28 +1,3 @@
-# To minimise change when upgrading to id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.15.6"
-# we have suppressed any formatting changes introduced by the updated version of this file, and instead
-# are using the formatting rules in place before this upgrade
-
-# Once the upgrade is complete, we can look at using the updated formatting rules, committed via a
-# separate PR. See https://dsdmoj.atlassian.net/browse/CAS-453. Once done, we should add this file
-# back into .gitignore
-
-[*.kt]
+[*.{kt,kts}]
```

Comment:
Ahh no I didn't.  I didn't realise the plugin would regenerate this file.  I've done this and amended this commit so that the content matches that created by the plugin when building.  Thanks.

Reply:
I guess we probably just want to remove this completely from the repository if we're happy with what the plugin is generating?

Reply:
Have added another commit to remove `.editorconfig` from the repo now.